cli/tests/unit/flash_test.ts

```diff
@@ -493,9 +448,10 @@ Deno.test(
   },
 );
 
+// TODO(bartlomieju): debug and fix
 // https://github.com/denoland/deno/issues/17291
 Deno.test(
-  { permissions: { net: true } },
+  { permissions: { net: true }, ignore: true },
```

Comment:
@littledivy please take a look

Reply:
It appears this test was broken as well - there's no problem pushing strings to a `ReadableStream`, unless it has `type: "bytes"`